webpack/app.js

```diff
@@ -157,9 +157,7 @@ module.exports = {
     unsafeCache: true
   },
 
-  node: {
-    fs: 'empty'
-  },
+  target: 'electron-renderer',
```

Comment:
Otherwise in `src/` I couldn't use modules that used `require('fs')` (the latter would return undefined), even though `window.require('fs')` worked fine in my `src/` files.

Not sure what the consequences of this are.

Reply:
No problem, the react app is exactly in the renderer process. I'll have a look at the CI error, building locally works fine.